ethexe/service/src/lib.rs

```diff
@@ -352,18 +346,12 @@ impl Service {
 
             log::trace!("Primary service produced event, start handling: {event:?}");
 
-            // Broadcast event.
-            // Never supposed to be Some in production.
-            if let Some(sender) = sender.as_ref() {
-                sender
-                    .send(event.clone())
-                    .expect("failed to broadcast service event");
-            }
+            #[cfg(test)]
```

Comment:
it wont work for integrational testing, please replace with some custom feature (passed as well for dev-deps)

Reply:
We don't have such case yet, so why not leave it as is?

Reply:
You moved tests from unit to integration ones

when you run `cargo test -p ethexe-service`, the code under `#[cfg(test)` isn't compiled for tests in ethexe/service/tests/mod.rs etc